x-pack/test/cloud_integration/config.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { resolve } from 'path';
+import { FtrConfigProviderContext } from '@kbn/test';
+import { services } from '../functional/services';
+import { pageObjects } from '../functional/page_objects';
+
+const FULLSTORY_ORG_ID = process.env.FULLSTORY_ORG_ID;
+const FULLSTORY_API_KEY = process.env.FULLSTORY_API_KEY;
+const RUN_FULLSTORY_TESTS = Boolean(FULLSTORY_ORG_ID && FULLSTORY_API_KEY);
```

Comment:
Note that I haven't added this config to any `test/scripts/test` scripts yet so this isn't running yet in CI. I'll do this in a follow up once I have these env vars configured in Jenkins. Still waiting to figure out which account we're going to be using for automated testing.

If you want to run this locally, ping me and I can send you the credentials.